app/controllers/concerns/ctc/can_begin_intake_concern.rb

```diff
@@ -9,13 +9,10 @@ module CanBeginIntakeConcern
     private
 
     def check_ctc_intake_cookie
-      return unless Rails.env.production?
+      return if open_for_ctc_intake?
```

Comment:
all the beta stuff gets handled in the open_for_intake? method already, so we can simplify this implementation a little.